pkg/languagedetection/privileged/privileged_detector_testutil.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
nit:
```suggestion
// Copyright 2024-present Datadog, Inc.
```

Reply:
I copied the same header as the file I copied this from, but I can definitely update this